.github/workflows/resetTrainingDB.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploy Training
+
+on:
+  workflow_dispatch:
```

Comment:
Was capturing some action items from our recent outage and came back to make sure this action can be triggered manually if needed.

Thank you for adding `on: workflow_dispatch` 😄